File: 2716/2716.c
int minimizedStringLength(char * s){
    int set[256] = {0}; // Initializing an array to simulate a set

    int length = 0;
    for (int i = 0; s[i] != '\0'; i++) {
        if (!set[s[i]]) {
            set[s[i]] = 1;
            length++;
        }
    }

    return length;
}

File: 2716/2716.php
class Solution {

    /**
    * @param String $s
    * @return Integer
    */
    function minimizedStringLength($s) {
        $uniqueChars = [];
        $length = 0;

        for ($i = 0; $i < strlen($s); $i++) {
            $char = $s[$i];
            if (!isset($uniqueChars[$char])) {
                $uniqueChars[$char] = true;
                $length++;
            }
        }

        return $length;
    }
}